layouts/index.html

```diff
@@ -7,9 +7,9 @@
     <a name="page-top"></a>
     {{ partial "nav.html" . }}
 
-    {{ with $.Site.Data.index }}
+    {{ with .Resources.ByType "hero" }}
       {{ range . }}
-        {{ $.Scratch.Set "data" . }}
+        <!-- {{ $.Scratch.Set "data" . }} -->
```

Comment:
Due to us using version control. We can just remove code without commenting out.

Reply:
Alright noted.